oauth2client/contrib/django_util/models.py

```diff
@@ -28,14 +28,14 @@ class CredentialsField(models.Field):
     """Django ORM field for storing OAuth2 Credentials."""
 
     def __init__(self, *args, **kwargs):
-        if 'null' not in kwargs:
-            kwargs['null'] = True
+        if "null" not in kwargs:
+            kwargs["null"] = True
         super(CredentialsField, self).__init__(*args, **kwargs)
 
     def get_internal_type(self):
-        return 'BinaryField'
+        return "BinaryField"
 
-    def from_db_value(self, value, expression, connection, context):
+    def from_db_value(self, value, expression, connection, context=None):
```

Comment:
Support for the context argument of Field.from_db_value() and Expression.convert_value() is removed.

https://docs.djangoproject.com/en/4.0/releases/3.0/#features-removed-in-3-0

Reply:
This appears to be the only non-formatting change in the PR 👍